Code/Core/FileIO/PathUtils.h

```diff
@@ -51,6 +51,9 @@ class PathUtils
     static void GetRelativePath( const AString & basePath,
                                  const AString & fileName,
                                  AString & outRelativeFileName );
+    static void GetBaseName( const AString& pathName, AString& baseName );
+    static void GetDirectoryName( const AString& pathName, AString& dirName );
+    static void JoinPath( AString& pathName, const AString& baseName );
```

Comment:
It would be nice to include some tests for these core functions.

I can help with those, but if you want to get familiar with the test framework (which we'll certainly want to utilize for the more complex functionality) then these trivial tests would be a good place to start.

Reply:
I will do it. Is there any documentation/description on how to run the tests?

Reply:
I sent tests for the the functions above as a separate followup commit. Let me know if this is something needs to be changed.